build.gradle

```diff
@@ -107,9 +107,12 @@ allprojects {
     }
 
     tasks.withType(JavaCompile) {
-        options.compilerArgs += ['-Werror']
-        // temporarily relax constraints until we can fix all violations
-        options.errorprone.disable 'AlmostJavadoc', 
+        options.errorprone {
+            disableWarningsInGeneratedCode = true
+            excludedPaths = '.*(generated).*'
```

Comment:
Is this necessary? I'm worried about paths that include a `generated` substring being ignored. I think this will replace the default excludedPaths configuration.

Reply:
when I have some cycles, I'll push some cleanup to the stream store renderer to allow removal of this (there's also a `Sets.newHashSet()` in the generated code.